### PR TITLE
Feat: sum

### PR DESCRIPTION
**Pull request by gijsvanwichen**

### CODE REVIEW

File: functions/sum/1.0/function.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "description": "Sum two numbers with an operator to sym two values into one number (eg, total is 'firstNumber' '+' 'secondNumber')",
+  "label": "Sum",
+  "category": "Misc",
+  "icon": "CreateIcon",
```
Comment (thomasvanophem) [inline functions/sum/1.0/function.json:5]: We need a different icon for this function

File: functions/sum/1.0/function.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+  "description": "Sum two numbers with an operator to sym two values into one number (eg, total is 'firstNumber' '+' 'secondNumber')",
+  "label": "Sum",
```
Comment (thomasvanophem) [inline functions/sum/1.0/function.json:3]: I think we should come up with a better name for this function seeing as we are not only doing a `sum`.

File: functions/sum/1.0/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    };
+  } else if (operator === "/") {
+    return {
+      result: left / right,
+    };
+  } else {
+    return {
+      result: left * right,
+    };
+  }
```
Comment (thomasvanophem) [inline functions/sum/1.0/index.js:18]: Perhaps we could move the `return` to the end of the function en declare a `result` variable. Something like
```
let result;

if (operator === "+") {
 result = left + right;
} ... 
else {
  result = left * right;
}

return {
  result,
};
```

**Review by thomasvanophem — APPROVED** [review]

File: functions/sum/1.0/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const sum = async ({ left, operator, right }) => {
+  if (operator === "+") {
```
Comment (justin-betty) [inline functions/sum/1.0/index.js:2]: If you use multiple `else if` statements it is nice to do a `switch` statement

**Comment by justin-betty** [comment]
Please add tests

**Pull request closed without merging**